build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 }
 
-version "0.4.0"
+version "0.4.0.1-custom"
```

Comment:
this can change back to what it was. ill update the version when i prepare it for a release.